### DISCUSSION TITLE
123pan 无法播放视频, 替代成官方的短片, 说123盘已启动防盗链功能

### DISCUSSION THREAD

**Discussion by syumayip** [AlistGo/alist#4632] [Q&A] [answered]
123pan 无法播放视频, 替代成官方的短片, 说123盘已启动防盗链功能

有知道怎么解决吗

**Comment by youtehub** [comment]
加密视频呗，[**alist-encrypt**](https://github.com/traceless/alist-encrypt),或者 [**rclone**](https://github.com/rclone/rclone) 在本地先加密后，再上传到网盘之中，然后在本地挂载一次解密播放

**Reply by syumayip** [reply]
谢谢，听上去很麻烦，还要先加密再上传

**Reply by pongfcnkl** [reply]
和是否加密无关，是把所有文件都改成防盗链视频了。

**Reply by syumayip** [reply]
大佬有解決方法嗎?
我登入網頁版, 視頻還是可以看的

**Comment by syumayip** [comment, marked as answer]
https://github.com/alist-org/alist/issues/4631
#4631 
這裡說官方已經關閉了, 剛才測試真的又能播放了
